### ISSUE TITLE
[jdbc] JdbcPostgresqlDAO uses wrong schema to retrieve all item tables

### ISSUE THREAD

**Issue by nimric** [openhab/openhab-addons#8889]
In the where clause of the sql statement `sqlGetItemTables` `table_schema='public'` is used. In my case the schema for all the item tables is a different one. I am not sure when the statment is executed, but you should be able to calculate all the table names since they are created earlier by the addon.

**Comment by nimric** [comment]
I checked the source code a bit and with the current implementation its not possible to retrieve the table names in another way. Therefore you should determine the correct schema from the `#itemsManageTable#` with a sub-select.
```
SELECT table_name FROM information_schema.tables WHERE table_type='BASE TABLE' AND table_schema=(SELECT table_schema
    FROM information_schema.tables WHERE table_type='BASE TABLE' AND table_name='#itemsManageTable#') AND NOT
    table_name='#itemsManageTable#'
```

**Comment by jlaur** [comment]
Fixed by #11023.
